k8s/k8s.go

```diff
@@ -186,7 +203,7 @@ func Instance() Ops {
 // Initialize the k8s client if uninitialized
 func (k *k8sOps) initK8sClient() error {
 	if k.client == nil {
-		k8sClient, err := getK8sClient()
+		k8sClient, snapClient, err := getK8sClient()
```

Comment:
why not have a separate method to get k8sSnapClient? currently it seems both are not used at the same time. We can call initSnapClient for snapshot methods. That way failures in initializing snapclient won't affect the k8s client.

Reply:
I think it's better to have just one function to initialize everything. Eventually the snapClient will go away once the snap changes in k8s are made upstream. So we will have to just change initK8sClient() instead of changing the init calls from Snapshot functions..